src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.EVP.Cipher.cs

```diff
@@ -115,8 +116,12 @@ internal static unsafe bool EvpCipherReset(SafeEvpCipherCtxHandle ctx, ReadOnlyS
 
         internal static void EvpCipherSetInputLength(SafeEvpCipherCtxHandle ctx, int inputLength)
         {
-            ref byte nullRef = ref MemoryMarshal.GetReference(Span<byte>.Empty);
-            if (!EvpCipherUpdate(ctx, ref nullRef, out _, ref nullRef, inputLength))
+            if (!EvpCipherUpdate(
+                ctx,
+                ref Unsafe.NullRef<byte>(),
```

Comment:
I wonder if it even makes sense to use `Unsafe.NullRef<T>()` internally now, given than `ref *(T*)null` can be used instead for all types 🤔

Reply:
According to #41418, `Unsafe.NullRef<T>()` is a "safe" API. What is the benefit to using `ref *(T*)null`?